### DISCUSSION TITLE
/pathtofile/config.yml ?

### DISCUSSION THREAD

**Discussion by MHamza-07** [wazuh/wazuh#13689] [Q&A] [answered]
Greetings,
I am trying to deploy certificate for my wazuh server.
According to **Certificates deployment** on
https://documentation.wazuh.com/current/user-manual/certificates.html
the file named 'config.yml'. is supposed to be edit, but I cannot find it.
I am using RHEL7.
![Wazuh](https://user-images.githubusercontent.com/85833913/171845342-2c19e32b-28dc-4404-90ec-e049dd5eb95e.png)
8

**Comment by mdiego92** [comment]
Hello MHamza-07,

The config.yml file you're looking for is the one you used during the installation. What version of Wazuh are you installing and what installation method are you using?

You could download the config.yml file with this command:

`curl -sO https://packages.wazuh.com/4.3/config.yml
`

Remember to set the configuration you need:

- If you're using an all-in-one deployment, you will need to set up set the <indexer-node-ip>, <wazuh-manager-ip>, and <dashboard-node-ip> parameters with 127.0.0.1.
- If you're using a distributed deployment you will need to set up the node names and IP values with the corresponding names and IP addresses for all Wazuh server, the Wazuh indexer, and the Wazuh dashboard nodes. Add as many node fields as needed.

Please let us know if you have any remaining questions.

Thank you very much.

**Reply by MHamza-07** [reply]
Hi mdiego92,

I'm working with wazuh version "v4.2.6".
and the deployment is all-in-one.
I tried to find the file config.yml and it gave me this as output:
![Capture](https://user-images.githubusercontent.com/85833913/172109244-e0becb43-26ef-43c8-935e-f6325ccbc9b7.PNG)

but the problem is, I don't know which file to edit.
Please guide.
kind regards.

**Comment by mdiego92** [comment, marked as answer]
Hello MHamza-07,

Since you're using Wazuh 4.2.6, you will need to use our documentation for **Wazuh 4.2**. In its configuration, you will use `instances.yml` instead of `config.yml`. If you followed our [step-by-step installation](https://documentation.wazuh.com/4.2/installation-guide/open-distro/all-in-one-deployment/all-in-one.html), you should have the instances.yml file as per our documentation:
![Screenshot from 2022-06-06 16-41-40](https://user-images.githubusercontent.com/62477966/172235576-49a758f6-0d45-4146-9d43-90d62b1387d6.png)
[https://documentation.wazuh.com/4.2/installation-guide/open-distro/all-in-one-deployment/all-in-one.html](https://documentation.wazuh.com/4.2/installation-guide/open-distro/all-in-one-deployment/all-in-one.html
)

Please let us know if you have any remaining questions.

Thank you very much.

**Reply by MHamza-07** [reply]
Brilliant !!!

Now I remember the file and was able to find it.
It was very helpful.
Thanks a bunch brother.
